### ISSUE TITLE
Create an AMP URL API example

### ISSUE THREAD

**Issue by sebastianbenz** [ampproject/amp-by-example#172]
Make it possible to play around with the API via sample JSON query in an editable textfield. Add a send button to retrieve and show the result. This needs to happen in an amp-iframe. 

//cc @kul3r4

**Comment by kul3r4** [comment]
I started at looking at this yesterday as part of #168 
Few things:
-we are not using GO templates, so it's harder to add new nodes into an html page. I was thinking to get all the content into a String, overwrite it, and then copy it back into the file
- we have the initial html page (under src) and the generated one (under dist). Changing the one under dist it's harder, given the gulp tasks add stuff on top of that. It would be easier edit the one under src and run the gulp task (compile) from the server (with a GO instruction). Do you think it makes sense?

**Comment by sebastianbenz** [comment]
We are already using Go templates so adding support for this isn't a problem: 

```
func getParameterDemoHandler(w http.ResponseWriter, r *http.Request) {
	value := r.URL.Query().Get("value")
	renderTemplate(w, "get-example", value)
}

func renderTemplate(w http.ResponseWriter, templateName string, value string) {
	t, _ := template.ParseFiles("templates/" + templateName + ".html")
	t.Execute(w, value)
}
```

**Comment by kul3r4** [comment]
Ok, but we are only using templates for generating new examples, once it's generated, it's difficult to change the example itself programmatically.
Let's take for example amp-img.html. At the moment it contains only a node for amp-img. If we want to add another node server side (implementing the new API), we will need to override the page completely because the page itself, amp-img.html does not contain any templating.

**Comment by sebastianbenz** [comment]
In the AMP URL sample you don't need server-side templating. You can
implement everything client-side in an iframe (you can't do it in AMP
anyway as you need html form support).

With amp-live-list it's different, I'd suggest hardcoding the route in the
backend to load and templatize the sample file. Once we need this more
often we can think about how to generalise this.

**Comment by sebastianbenz** [comment]
Implemented in the following PRs: #181 & #189.
